asset_dashboard/static/js/components/maps/SelectAssetsMap.js

```diff
@@ -79,14 +81,11 @@ function SelectAssetsMap(props) {
         body: JSON.stringify(data)
     }).then((response) => {
       if (response.status == 201) {
-        // TODO: this reloads the page but clears the user search...
-        // Need to come up with way to reload by rehydrating the previous state
+        setAjaxMessage('Assets successfully saved.')
```

Comment:
We talked about not having the page reload after rendering a message. I understand we're using the page reload to update the local asset list serverside. What about using the browser session state via the rook library, like you did to persist the user's last search, to store a success message, then render and remove it from the state when the page reloads?

Reply:
@hancush Ah, I guess I didn't explain what I was talking about well enough / we we're thinking two different things. I didn't like how the message persisted **after** the page reloads. I thought it makes sense that the message shows up after the ajax call, and then the page reloads, and then the message is gone once the page reloads, like how it works now. 

I can use session storage, but I think that might add some extra complexity to clear it after a page reload. I'll have to check that the message is there every time the component mounts, and then show it, and then clear it from sessionStorage once the user exits out of it.

Reply:
I'm seeing now that the reload is a lot quicker in the deployment so my use case doesn't make as much sense. It takes a long time to reload in development so it seemed like that was enough time to show the message. But I think that you're right and session storage messages would be the best way to go.

I thought when you were saying "save that message to the session" you meant [the django session](https://docs.djangoproject.com/en/4.0/topics/http/sessions/) and not session storage.

Reply:
I wondered if the session mixup was the case, @smcalilly! 😓 Have what you need to make that change?